README.md

```diff
@@ -19,50 +19,70 @@ The OSHDB offers a distributed data base that splits storage and computation loa
+* _High Performance_: The OSM history data set is large and thus requires efficiency in the way the data is stored and in the way it can be accessed and processed. 
+* _Local and Distributed Deployment_: Analysis queries should scale well from data explorations of small regions up to global studies of the complete OSM data set.
+
+### Data Model
+
+The OSHDB uses it's own binary data format that encapsulates the OSM history data available from [planet.osm.org](https://planet.osm.org/) and is optimized for efficiently storage and access to the history data. In order to allow parallelized data processing, the OSHDB data model also includes a data partitioning scheme.
 
-* *(close to osm data structures)*
-* *(no lossy transformations/compressions)*
-* *(partially resolved references, to speed up typical operations like "geometry building")*
-* *(split into cells)*
+[![schematic overview of the OSHDB data model](documentation/manual/data-model.svg)](documentation/manual/data-model.md)
 
-### Data Processing/Access
+See the [data model](documentation/manual/data-model.md) section of the documentation for further information about the OSHDB data model.
 
-* *(rest-based api for common queries)*
-* *(oshdb-api: flexible, simple, map-reudce pattern, works on actual geometries)*
-* *(raw-data api: direct access to underlying raw osm data)*
+### API
+
+The OSHDB offers a flexible, simple and intuitive application programming interface that provides helpful abstractions on top of the offered OSM data entities. It provides different _views_ on the OSM history data that allow to either investigate the OSM data at specific points in time (as snapshots), or to investigate all contributions to the OSM data in their entirety. The OSHDB API also allows to filter the OSM data by arbitrary regions, time ranges and OSM properties such as tags, entity type, etc.
+
+The API is based on the MapReduce programming model, and offers powerful methods to aggregate and analyze the OSM history data. The OSHDB API is described in detail in the [api section](documentation/manual/api.md) of the documentation.
 
 Installation
 ------------
 
-*(install instructions: maven, link to separate developer instructions)*
+The OSHDB is available as a pre-compiled maven library and can be incorporated easily in any maven project. If you're starting a new project, take a look at how your IDE handles maven projects (for example, here you find instructions how to create a new maven project using [IntelliJ](https://www.jetbrains.com/help/idea/maven-support.html#maven_create_project)). Our [first steps tutorial](https://github.com/GIScience/oshdb/tree/docu-update-cleanup/documentation/first-steps#2-add-maven-dependency) includes further information about how to add the OSHDB as a maven dependency to your projects.
 
-API
+Documentation
```

Comment:
put "Table of contents" to top, so people can get hands-on (with the tutorial) without having to scroll down here

- add "introduction" (=this page) to toc

Reply:
I put a mention of the tutorial at the beginning (after the first code example) of the main readme. 915f8600

not sure if a toc would be needed regardless. :thinking:

Reply:
I guess it would look organised but is not compulsory :-)